app/views/orders/order_cancel.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="d-flex justify-content-center align-items-center">
+  <div class="col-md-4">
+    <div class="border border-3 border-danger"></div>
+    <div class="card  bg-white shadow p-5">
+      <div class="mb-4 text-center">
+        <svg xmlns="http://www.w3.org/2000/svg" class="text-danger" width="75" height="75" fill="currentColor" class="bi bi-x-circle-fill" viewBox="0 0 16 16">
+          <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM5.354 4.646a.5.5 0 1 0-.708.708L7.293 8l-2.647 2.646a.5.5 0 0 0 .708.708L8 8.707l2.646 2.647a.5.5 0 0 0 .708-.708L8.707 8l2.647-2.646a.5.5 0 0 0-.708-.708L8 7.293 5.354 4.646z"/>
+        </svg>
+      </div>
+      <div class="text-center">
+        <h1>Sorry !</h1>
+        <p>Your payment was unsuccessful</p>
+        <%= link_to 'Cart', cart_path, class: 'btn btn-outline-danger mx-2' %>
+        <%= link_to 'Back Home', root_path, class: 'btn btn-outline-danger mx-2' %>
+        <div class='mt-3'>
+          <h5>Do not refresh the page</h5>
+          Redirecting to your cart in <span id="countdown">5</span> seconds...
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<script>
+  var countdownElement = document.getElementById('countdown');
```

Comment:
Move all the scripts from the html to different files under assets folder and import respective files in html files,

Reply:
Do it at the end. Complete the project first and then move all of them

Reply:
Ok sir